### ISSUE TITLE
GitHub OIDC is missing attribute_condition

### ISSUE THREAD

**Issue by neotrow** [terraform-google-modules/terraform-google-github-actions-runners#156]
### TL;DR

We are getting `New Advisory Notification` in GCP because there is no attribute_condition set:

 Would be great if the Example Usage in the Readme: https://github.com/terraform-google-modules/terraform-google-github-actions-runners/tree/master/modules/gh-oidc would include this. 

### Expected behavior

The Readme https://github.com/terraform-google-modules/terraform-google-github-actions-runners/tree/master/modules/gh-oidc should include the attribute_condition in the examples.

Or go even further and make attribute_condition a required field. 

### Observed behavior

We are getting `Important information about Workload Identity Federation and Identity Pool Configurations` notifcations in GCP because we are missing an attribute_condition. 

### Terraform Configuration

```hcl
module "gh_oidc" {
  source      = "terraform-google-modules/github-actions-runners/google//modules/gh-oidc"
  project_id  = var.project_id
  pool_id     = "example-pool"
  provider_id = "example-gh-provider"
  sa_mapping = {
    "foo-service-account" = {
      sa_name   = "projects/my-project/serviceAccounts/foo-service-account@my-project.iam.gserviceaccount.com"
      attribute = "attribute.repository/${USER/ORG}/<repo>"
    }
  }
}
```
```


### Terraform Version
irrelevant


### Additional information

_No response_

**Comment by madi-parloa** [comment]
I'm getting the same problem I think when this module is used in teraform-exanple-foundation repo [here](https://github.com/terraform-google-modules/terraform-example-foundation/blob/0bc707bc665d31ca9d20f8e984e780bf656d99e4/0-bootstrap/github.tf.example#L92):
```
│ Error: Error creating WorkloadIdentityPoolProvider: googleapi: Error 400: The attribute condition must reference one of the provider's claims. For more information, see https://cloud.google.com/iam/docs/workload-identity-federation-with-deployment-pipelines#conditions
│ 
│   with module.gh_oidc.google_iam_workload_identity_pool_provider.main,
│   on .terraform/modules/gh_oidc/modules/gh-oidc/main.tf line 26, in r
```
